pages/common/pixiecore.md

```diff
@@ -0,0 +1,8 @@
+# pixiecore
+
+> A tool to manage network booting of machines.
+> More information: <https://github.com/danderson/netboot/tree/master/pixiecore>.
+
+- Start a PXE boot server which provides a `netboot.xyz` boot image:
```

Comment:
Maybe it could be even more clarified.
```suggestion
- Start a PXE boot server which provides a boot image from the `netboot.xyz` server:
```

Reply:
netboot.xyz is really one image. Not sure this would quite fit?

Reply:
Then I'd keep the original. I think, even if you misinterpret `netboot.xyz` as a file, it wouldn't be that bad.

Reply:
Oh I see. Definitely ignore this then.